wsf_scraping/pipelines.py

```diff
@@ -128,18 +122,20 @@ def check_keywords(self, item, spider_name, base_pdf_path):
     def process_item(self, item, spider):
         """Process items sent by the spider."""
 
-        base_pdf_path = os.path.join(tempfile.gettempdir(), item['pdf'])
-        file_hash = get_file_hash(base_pdf_path)
+        if not item['pdf']:
```

Comment:
will add an issue to discuss whether 'pdf' is the right name since it was unclear to me that this was the pdf file name, i would expect it to be the content of the scraped pdf. not a priority atm but worth thinking whether the name is the right one.